packages/commonwealth/server/routes/deleteChain.ts

```diff
@@ -251,6 +252,14 @@ const deleteChain = async (
     );
 
     // TODO: Remove this once we figure out a better way to relate addresses across many chains (token communities)
+    await models.sequelize.query(
```

Comment:
Can we do this all with Sequelize instead of raw SQL?

Reply:
Want the whole route changed to match or just these?

Reply:
Ideally the whole route.

Reply:
okay